x-pack/plugin/inference/qa/test-service-plugin/src/main/java/org/elasticsearch/xpack/inference/mock/TestSparseInferenceServiceExtension.java

```diff
@@ -121,23 +121,25 @@ private SparseEmbeddingResults makeResults(List<String> input) {
             for (int i = 0; i < input.size(); i++) {
                 var tokens = new ArrayList<SparseEmbeddingResults.WeightedToken>();
                 for (int j = 0; j < 5; j++) {
-                    tokens.add(new SparseEmbeddingResults.WeightedToken(Integer.toString(j), (float) j));
+                    tokens.add(new SparseEmbeddingResults.WeightedToken("feature_" + j, j + 1.0F));
```

Comment:
Changes on the test inference service to return results that can actually be indexed

Reply:
My bad - I changed that for readability sake, nothing else. Numeric strings can be indexed.

Reply:
> Changes on the test inference service to return results that can actually be indexed

This comment is what caused me to question this. But I see that you are also now adding `1` to the value as we don't allow a `0`.

This is ok.